dpnp/backend/extensions/blas/syrk.cpp

```diff
@@ -245,42 +336,43 @@ std::pair<sycl::event, sycl::event>
     src_nelems = n * n;
 #endif // USE_ONEMATH_CUBLAS
 
-    const std::int64_t lda = is_row_major ? k : n;
-    const std::int64_t ldc = n;
-    dpctl::tensor::validation::CheckWritable::throw_if_not_writable(resultC);
-    dpctl::tensor::validation::AmpleMemory::throw_if_not_ample(resultC,
-                                                               src_nelems);
-
-    const int matrixA_typenum = matrixA.get_typenum();
-    const int resultC_typenum = resultC.get_typenum();
-    if (matrixA_typenum != resultC_typenum) {
-        throw py::value_error("Given arrays must be of the same type.");
-    }
+        const std::int64_t lda = is_row_major ? k : n;
+        const std::int64_t ldc = n;
+        dpctl::tensor::validation::CheckWritable::throw_if_not_writable(
+            resultC);
+        dpctl::tensor::validation::AmpleMemory::throw_if_not_ample(resultC,
+                                                                   src_nelems);
+
+        const int matrixA_typenum = matrixA.get_typenum();
+        const int resultC_typenum = resultC.get_typenum();
+        if (matrixA_typenum != resultC_typenum) {
+            throw py::value_error("Given arrays must be of the same type.");
+        }
 
-    auto array_types = dpctl_td_ns::usm_ndarray_types();
-    const int type_id = array_types.typenum_to_lookup_id(matrixA_typenum);
-    syrk_impl_fn_ptr_t syrk_fn = syrk_dispatch_vector[type_id];
-    if (syrk_fn == nullptr) {
-        throw py::value_error(
-            "No syrk implementation is available for the specified data type "
-            "of the input and output arrays.");
-    }
+        auto array_types = dpctl_td_ns::usm_ndarray_types();
+        const int type_id = array_types.typenum_to_lookup_id(matrixA_typenum);
+        syrk_impl_fn_ptr_t syrk_fn = syrk_dispatch_vector[type_id];
+        if (syrk_fn == nullptr) {
+            throw py::value_error("No syrk implementation is available for the "
+                                  "specified data type "
+                                  "of the input and output arrays.");
+        }
 
-    const char *a_typeless_ptr = matrixA.get_data();
-    char *r_typeless_ptr = resultC.get_data();
+        const char *a_typeless_ptr = matrixA.get_data();
+        char *r_typeless_ptr = resultC.get_data();
 
 #if defined(USE_ONEMATH_CUBLAS)
-    sycl::event syrk_ev = syrk_fn(exec_q, transA, n, k, a_typeless_ptr, lda,
-                                  r_typeless_ptr, ldc, depends);
+        sycl::event syrk_ev = syrk_fn(exec_q, transA, n, k, a_typeless_ptr, lda,
+                                      r_typeless_ptr, ldc, depends);
 #else
     sycl::event syrk_ev = syrk_fn(exec_q, transA, n, k, a_typeless_ptr, lda,
                                   r_typeless_ptr, ldc, is_row_major, depends);
```

Comment:
Can we just pass `is_row_major = false` in case of `defined(USE_ONEMATH_CUBLAS)`?
It's quite annoying to wrap every nested function call with `#if defined(USE_ONEMATH_CUBLAS)`.